pkgs/top-level/all-packages.nix

```diff
@@ -1350,6 +1350,10 @@ in
 
   fd = callPackage ../tools/misc/fd { };
 
+  fdroidserver = callPackage ../development/tools/fdroidserver {
+    pythonPackages = python3Packages;
```

Comment:
```suggestion
```

Reply:
Without this, Python 2 is chosen and docker-py complains because it doesn't run on 2.7.